src/utils/vectors.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports.isNullVector = function (vector) {
```

Comment:
You could put this in `utils/coordinates.js` where all the other vector stuff is.